src/runtime_src/core/pcie/linux/pcidev.h

```diff
@@ -142,43 +129,93 @@ class pci_device
   get_sysfs_path(const std::string& subdev, const std::string& entry);
 
   virtual std::string
-  get_subdev_path(const std::string& subdev, uint32_t idx);
-  virtual int pcieBarRead(uint64_t offset, void *buf, uint64_t len);
-  virtual int pcieBarWrite(uint64_t offset, const void *buf, uint64_t len);
-
-  virtual int open(const std::string& subdev, int flag);
-  virtual int open(const std::string& subdev, uint32_t idx, int flag);
-  void close(int devhdl);
-  int ioctl(int devhdl, unsigned long cmd, void *arg = nullptr);
-  int poll(int devhdl, short events, int timeoutMilliSec);
-  virtual void *mmap(int devhdl, size_t len, int prot, int flags, off_t offset);
-  virtual int munmap(int devhdl, void* addr, size_t len);
-  int flock(int devhdl, int op);
-  int get_partinfo(std::vector<std::string>& info, void *blob = nullptr);
-  std::shared_ptr<pcidev::pci_device> lookup_peer_dev();
+  get_subdev_path(const std::string& subdev, uint32_t idx) const;
+
+  virtual int
+  pcieBarRead(uint64_t offset, void *buf, uint64_t len);
+
+  virtual int
+  pcieBarWrite(uint64_t offset, const void *buf, uint64_t len);
+
+  virtual int
+  open(const std::string& subdev, int flag) const;
+
+  virtual int
+  open(const std::string& subdev, uint32_t idx, int flag) const;
+
+  virtual void
+  close(int devhdl) const;
+
+  virtual int
+  ioctl(int devhdl, unsigned long cmd, void *arg = nullptr) const;
+
+  virtual int
+  poll(int devhdl, short events, int timeoutMilliSec);
```

Comment:
```suggestion
  poll(int devhdl, short events, int timeout_ms);
```
No camelCase, please.

Reply:
Not my code, but I can clean it up.